Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -743,7 +756,7 @@
                 % pvPrimary is the main tissue type that is investigated
                 % pvSecondary is the other tissue type that is used with PVC
 
-				if x.S.bMasking(3)==0 % no tissue-masking
+				if bMasking(3)==0 % no tissue-masking

```

Comment:
For consistency, we can change these statements throughout into:

if bMasking

and 

if ~bMasking